fee_calculator/apps/api/serializers.py

```diff
@@ -1,12 +1,58 @@
 # -*- coding: utf-8 -*-
 from django.shortcuts import get_object_or_404
+from django.db.utils import OperationalError
 from rest_framework import serializers
 
 from calculator.models import (
     Scheme, Scenario, FeeType, AdvocateType, OffenceClass, Price, Unit,
     ModifierType, Modifier, ScenarioCode
 )
 
+from calculator import models as calc_models
```

Comment:
I don't think this import is needed. It looks like it was for doing `calc_models.Unit` in the other file, but when you brought it across you already had `Unit` imported so correctly switched to using that, so have no use for `calc_models` itself now

Reply:
Good spot. Removed.